content/en/docs/How Tos/pipeline_configuration.md

```diff
@@ -83,6 +103,10 @@ can be specified, and in this case we simply provide the files option to List.
 Notice that List would accept http:// or even s3:// URLs there in addition to local paths,  
 and some more (run ./Baker-bin -help List in the help example for more details).
 
+`[filterchain]` defines the configuration for the whole filter chain. Filter-specific configuration
+are provided by `[[filter]]` (see below). The only accepted configuration is `procs = <int>` that
```

Comment:
Either:
 - Filter-specific configuration is provided
 - Filter-specific configurations are provided

I guess